apps/2i/hotel/frontend/src/app/admin/_Components/Amenities.tsx

```
@@ -0,0 +1,38 @@
+import { Button } from '@/components/ui/button';
+import { Card } from '@/components/ui/card';
+import { Dialog, DialogContent, DialogDescription, DialogHeader, DialogTitle, DialogTrigger } from '@/components/ui/dialog';
```

ESLint

[error] 'DialogDescription' is defined but never used.

(no-unused-vars)